portfolio/src/main/webapp/script.js

```diff
@@ -156,31 +159,50 @@ function getComments() {
 }
 
 /**
- * Create element that represents a comment.
+ * Format timestamp for display in comment section.
  */
-function createCommentElement(comment) {
-  const commentElement = document.createElement('div');
-
-  const nameElement = document.createElement('div');
-  nameElement.innerText = comment.name;
-
-  const textElement = document.createElement('div');
-  textElement.innerText = comment.text;
-
-  const timeElement = document.createElement('div');
-  const timestampString = new Date(comment.timestamp).toString();
-  timeElement.innerText = timestampString;
-
-  const scoreElement = document.createElement('div');
-  scoreElement.innerText = 'Sentiment score (Scale from -1 to 1): ' +
-      comment.sentimentScore.toString();
+function parseTimestamp(timestamp) {
+  const date = new Date(timestamp);
+  return date.toLocaleString();
+}
 
-  commentElement.appendChild(nameElement);
-  commentElement.appendChild(textElement);
-  commentElement.appendChild(timeElement);
-  commentElement.appendChild(scoreElement);
+/**
+ * Get name of icon that represents sentiment score.
+ */
+function getIcon(sentimentScore) {
+  /* Conditions are ordered. */
+  if (sentimentScore >= 0.5) {
+    return 'sentiment_very_satisfied';
+  }
+  else if (sentimentScore >= 0) {
+    return 'sentiment_satisfied';
+  }
+  else if(sentimentScore >= -0.5) {
+    return 'sentiment_dissatisfied';
+  }
+  else {
+    return 'sentiment_very_dissatisfied';
+  }
+}
 
-  return commentElement;
+/**
+ * Create element that represents a comment.
+ */
+function getCommentHtml(comment) {
+  const tooltipUrl = 'https://cloud.google.com/natural-language/docs/basics' +
+      '#interpreting_sentiment_analysis_values';
+
+  const html = `<li><div class="comment"><div>
```

Comment:
you should probably format this to look like it would have if you put in the html file. something like
```
`
<li>
  <div class="comment">
    <div>
      <div class="comment-name">${comment.name}</div>
      ...
    </div>
</li>
`
```